### PR TITLE
Adding note about new behavior when cleaning resources

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/27025

### CODE REVIEW

File: governance/create_policy.adoc [hunk 1]
```diff
@@ -309,4 +309,8 @@ Use the `pruneObjectBehavior` parameter in a configuration policy to clean up re
 
 You can set the value directly in the YAML as you create a policy from the CLI. From the console, you can select the value in the _Prune Object Behavior_ section of the the _Policy templates_ step.
 
-*Note:* If a policy that installs an operator uses the `pruneObjectBehavior` parameter defined, then additional clean up is needed to complete the operator uninstall. Additional clean up might include deleting the operator `ClusterServiceVersion` object.
+*Notes:*
+
+- If a policy that installs an operator uses the `pruneObjectBehavior` parameter defined, then additional clean up is needed to complete the operator uninstall. Additional clean up might include deleting the operator `ClusterServiceVersion` object.
+
+- As you disable the `config-policy-addon` and the Kubernetes configuration policy custom resource is deleted on the managed cluster, you must first remove the resources from the managed cluster.
```
Comment (JustinKuli) [inline governance/create_policy.adoc]: This makes it sound like you need to remove the resources before the addon can be disabled, which isn't quite right.

When the addon is disabled, the `pruneObjectBehavior` is ignored, and related resources are not cleaned up automatically. If the user wanted the resources deleted, they could either remove the policies from the cluster first (which would trigger the cleanup), or they could manually remove the resources after the addon is disabled.

I think we would suggest the first option here, something like:

```suggestion
- As you disable the `config-policy-addon` on the managed cluster, the `pruneOjbectBehavior` is ignored. To automatically clean up the related resources on the policies, the policies should be removed from the managed cluster before the addon is disabled.
```

**Review by JustinKuli — CHANGES_REQUESTED** [review]

File: governance/create_policy.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/create_policy.adoc]: @JustinKuli Thanks for your suggestion! What are your thoughts about this update:

As you disable the `config-policy-addon` on the managed cluster, the `pruneOjbectBehavior` is ignored. To automatically clean up the related resources on the policies, you must remove the policies from the managed cluster before the add-on is disabled.

File: governance/create_policy.adoc [hunk 2]
```diff
@@ -309,4 +309,8 @@ Use the `pruneObjectBehavior` parameter in a configuration policy to clean up re
 
 You can set the value directly in the YAML as you create a policy from the CLI. From the console, you can select the value in the _Prune Object Behavior_ section of the the _Policy templates_ step.
 
-*Note:* If a policy that installs an operator uses the `pruneObjectBehavior` parameter defined, then additional clean up is needed to complete the operator uninstall. Additional clean up might include deleting the operator `ClusterServiceVersion` object.
+*Notes:*
+
+- If a policy that installs an operator uses the `pruneObjectBehavior` parameter defined, then additional clean up is needed to complete the operator uninstall. Additional clean up might include deleting the operator `ClusterServiceVersion` object.
```
Comment (cadawson) [inline governance/create_policy.adoc]: This reads a little strangely. Alternate suggestion: If a policy that installs an operator has the `pruneObjectBehavior` parameter defined, additional cleanup is needed to complete the operator uninstallation. You might need to delete the operator `ClusterServiceVersion` object as part of this cleanup.

File: governance/create_policy.adoc (hunk 1 above)
Comment (cadawson) [reply governance/create_policy.adoc]: Typo in pruneObjectBehavior

**Review by cadawson — APPROVED** [review]
Suggestion added, otherwise looks good.

File: governance/create_policy.adoc (hunk 1 above)
Comment (JustinKuli) [reply governance/create_policy.adoc]: Typo is my fault! Yes, that sounds better

File: governance/create_policy.adoc (hunk 2 above)
Comment (JustinKuli) [reply governance/create_policy.adoc]: I like that

File: governance/create_policy.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/create_policy.adoc]: Updated

**Comment by dockerymick** [comment]
@cadawson @JustinKuli thanks for the reviews and suggestions. I have updated the PR. When you have an opportunity, will you re-review?

**Review by JustinKuli — APPROVED** [review]
LGTM

**Comment by dockerymick** [comment]
Thanks to you both! Merging this PR

**Pull request merged**